marketplace_builder/custom_model_types/application_files.yml

```diff
@@ -1,6 +1,4 @@
 name: application_files
-instance_profile_types:
-  - developer
```

Comment:
What implications does it have?
https://github.com/mdyd-dev/marketplace-nearme-example/blob/22b71a471c6b079ea7764ed37cb52ae6caad7c13/marketplace_builder/form_configurations/developer/upload_application.liquid#L8-L18

This code looks like its using this relation - does it work after this change? (i think we dont have e2e test for that yet)